contrib/kvm-setup/roles/kvm-setup/tasks/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+
+- name: Update apt
+  raw: apt-get update
+  when: bootstrap_os in ['Ubuntu', 'Debian']
+
+- name: Basic packages required by ansible
+  raw: apt-get install -y python python-apt
+  when: bootstrap_os in ['Ubuntu', 'Debian']
+
+- name: Set python interpeter
+  set_fact:
+    ansible_python_interpreter: "/usr/bin/python"
+
+- name: Gather facts
+  setup:
+
+- name: Upgrade all packages to the latest version
+  command: DEBIAN_FRONTEND=noninteractive apt-get -o \
+       Dpkg::Options::=--force-confdef -o \
+       Dpkg::Options::=--force-confold -q -y \
+       dist-upgrade --allow-downgrades
```

Comment:
well if you intentionally put high package pins, it does it. by default, you don't have them.. but maybe still too risky to automate